simsx/runner.go

```
@@ -110,6 +110,27 @@
+		db corestore.KVStoreWithBatch,
+		traceStore io.Writer,
+		loadLatest bool,
+		appOpts servertypes.AppOptions,
+		baseAppOptions ...func(*baseapp.BaseApp),
+	) T,
+	setupStateFactory func(app T) SimStateFactory,
+	seeds []int64,
+	fuzzSeed []byte,
+	randAccFn simtypes.RandomAccountFn,
+	postRunActions ...func(t testing.TB, app TestInstance[T], accs []simtypes.Account),
+
```

golangci-lint (1.62.2)

File is not `gofumpt`-ed with `-extra`

(gofumpt)